lib/affiliate_credits.rb

```diff
@@ -39,17 +42,29 @@ def check_affiliate
   end
 
   def check_affiliate_internal(ref_id, user)
+    return commission_referral_create_credit(user) if use_commission_referral?(user)
```

Comment:
I tested this but even if I'm on commissions, the accounts that I refer end up being added to the `spree_affiliates` table. I think we have to check if commission is enabled in the wallet, and that this is the place where we want to actually save the referred account to the `commission_referrals` table if the referrer is on commissions, right?

Reply:
thanks maida, moving back to development to check it.